packages/devextreme/testing/tests/DevExpress.exporter/exceljsParts/exceljs.dataGrid.tests.js

```diff
@@ -40,6 +41,7 @@ const moduleConfig = {
     beforeEach: function() {
         this.worksheet = new ExcelJS.Workbook().addWorksheet('Test sheet');
         this.customizeCellCallCount = 0;
+        setLicenseCheckSkipCondition();
```

Comment:
Because this test also checks the number of calls of the errors.log method and the new functionality interferes with this test.

Reply:
I suggest you to make the actual test more concrete instead of adding of additional actions.
```javascript
        this.stub = sinon.stub(errors, 'log').callsFake((errorId) => {
            if(errorId === 'W0003') {
                QUnit.assert.strictEqual(true, false, 'no deprecation warnings should be logged');
            }
        });
```